tests/unittest/_torch/attention/kv_cache_utils.py

```
@@ -0,0 +1,142 @@
+    if all(k.shape[0] == 0 for k in k_per_seq):
+        # Pure prefill (nothing cached): the backend populates the cache via its
+        # own append, so no manual fill is needed. An fp8 pool must still be
+        # zeroed, though -- the context kernel reads whole blocks and an
+        # uninitialized e4m3 byte decodes to NaN, poisoning the softmax (a real
+        # run never hits this; its kernel appends incrementally). NVFP4 pools are
+        # left as-is: get_buffers cannot reshape the half-size packed pool.
+        try:
+            buf = kv_cache_manager.get_buffers(layer_idx, kv_layout=kv_layout)
+        except TypeError:
+            buf = kv_cache_manager.get_buffers(layer_idx)
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)